From d1797e9005cc9787e7a3d7277ef08dc3b635d52e Mon Sep 17 00:00:00 2001
From: zhongjie <zhongjie.shi@intel.com>
Date: Wed, 12 Dec 2018 15:50:34 +0800
Subject: [PATCH 2/6] Revert "primary audio hal" to enable KOG.

This reverts commit 457663d87f1d77207895c56580b6fa7613852aba.

Change-Id: I4c1d7e67ae57d27cb903bf33be35ae7b14446ea1
Tracked-On:
Signed-off-by: zhongjie <zhongjie.shi@intel.com>
Reviewed-on: 655142
Reviewed-by: Xue, Jin <jin.xue@intel.com>
Tested-by: Xue, Jin <jin.xue@intel.com>
Reviewed-by: Zhang, Hongyu <hongyu.zhang@intel.com>
Reviewed-on: 657226
Tested-by: Zhang, Hongyu <hongyu.zhang@intel.com>
---
 Android.mk | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index ccf2807..68c26bf 100644
--- a/Android.mk
+++ b/Android.mk
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+ifeq ($(INTEL_AUDIO_HAL),audio)
+
 LOCAL_PATH := $(call my-dir)
 
 include $(CLEAR_VARS)
@@ -33,10 +35,15 @@ LOCAL_C_INCLUDES += \
 	$(call include-path-for, audio-route) \
 	$(call include-path-for, audio-effects)
 
-LOCAL_MODULE := audio.primary.$(TARGET_PRODUCT)
+#Preferred paths for all vendor hals /vendor/lib/hw
+LOCAL_PROPRIETARY_MODULE := true
+
+LOCAL_MODULE := audio.primary.$(TARGET_BOARD_PLATFORM)
 
 LOCAL_MODULE_RELATIVE_PATH := hw
 
 LOCAL_MODULE_TAGS := optional
 
 include $(BUILD_SHARED_LIBRARY)
+
+endif # INTEL_AUDIO_HAL
-- 
2.7.4

